### DISCUSSION TITLE
Can I install Necro/Third-Party Apps?

### DISCUSSION THREAD

**Discussion by MoirDevelops** [whs/runekit#43] [Q&A macOS]
I'm trying to install https://github.com/NadyaNayme/NyusNecroJobGauge to my Macs RuneKit. 

The .json file that is mentioned to install it is, alt1://addapp/https://nadyanayme.github.io/NyusNecroJobGauge/dist/appconfig.json

I then receive this error https://snipboard.io/LpEzaq.jpg

Not sure what to try next. Any help is appreciated! I am running Sonoma 14.3.1 (23D60)

**Comment by schoentoon** [comment]
After launching runekit, you should go to the settings of runekit (right click the app tray icon, then settings). In this new window simply press the plus button. Paste the link to the appconfig.json here (without alt1://addapp/). That should do the trick.

**Reply by MoirDevelops** [reply]
Thank you, this worked! Now I'm running into the issue of RuneKit not being able to find my buff/debuff bars as well as AFK warden not being able to see my screen. 

I have given permission to the following,
- Accessibility - for access to game window
<img width="485" alt="image" src="https://github.com/whs/runekit/assets/105198024/0934dd10-efdf-4ee0-807d-aed2f6ebfad3">

- Input Monitoring - for hooking alt+1 and idle detection
<img width="488" alt="image" src="https://github.com/whs/runekit/assets/105198024/705f94e0-a8a3-43ac-a081-59e621816eea">

- Screen Recording - for capturing game
<img width="495" alt="image" src="https://github.com/whs/runekit/assets/105198024/596aa47c-6632-4eae-8454-7318f805d229">

Hope I can figure this out. Any help is appreciated.

**Reply by schoentoon** [reply]
Don't think I can help you any further with that sadly. I run runekit on Linux.
